_data/finance.yml

```diff
@@ -9,6 +9,13 @@ websites:
     facebook: AutomaticDataProcessing
     img: adp.png
 
+  - name: Allianz
+    url: https://www.allianz.de/
+    img: allianz.png
+    tfa:
+      - sms
```

Comment:
Also supports iTAN
```suggestion
      - sms
      - hardware
```

Reply:
I'm not quite sure about this, the documentation is kind of dodgy there. One paragraph only talks about mTAN (= SMS) as the second factor, but another mentions iTAN for authentication.
I don't have an Allianz account myself, but I could send them a mail for clarification.

Anyway, since iTAN is only a paper-based list of codes, this doesn't even qualify as a second authentication factor in my opinion. It falls under "knowledge" as well, doesn't it?

Reply:
Paper codes are hardware 2FA. At least to our definition.

Reply:
Okay, I will contact Allianz support to provide clarification in this matter then.

Reply:
Still no reply by Allianz support to my mail from 2019-10-26... :frowning_face:
I wonder if they monitor their email at all.

Reply:
Meh, let's skip this then. Thanks for trying :+1:

Reply:
Just to bring closure to this cliffhanger:
The Allianz support finally replied and confirmed that only SMS is supported for 2FA.